gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started.adoc

```diff
@@ -4,16 +4,169 @@ include::_attributes.adoc[]
+
+* A username and password for your Anypoint Platform organization. If you don’t have Anypoint Platform yet, https://anypoint.mulesoft.com/login/signup[create a Trial organization^] on Anypoint Platform.
+
+* A machine with an x86 processor. M1 and M2 processors are not supported.
+
+* A Docker installation. See https://www.docker.com/products/docker-desktop/[Docker Desktop^] for an example.
+
+== Install Flex Gateway
+
+To install the Flex Gateway Docker image, run the following command in a terminal:
+
+[source,ssh]
+----
+docker pull mulesoft/flex-gateway:latest
+----
+
+[NOTE]
+Use `sudo` if you encounter file permission issues when running this command.
+
+== Register Flex Gateway
+
+. Create a new directory by running the following command:
++
+[source,ssh]
+----
+mkdir flex-config
+----
+
+. Navigate to the new directory by running the following command:
++
+[source,ssh]
+----
+cd flex-config
+----
+
+. Log in to https://anypoint.mulesoft.com/login/[Anypoint Platform^] with your Anypoint username and password.
+. Select *Runtime Manager*.
+. Click on the *Flex Gateways* tab in the side navigation panel.
+. Click *Add Gateway*.
+. Select *Docker*.
+. Copy the code block in the *Register your gateway* section. The code block will contain your unique organization ID and token.
++
+The following screenshot of the *Add a Flex Gateway* page highlights the code block that should be copied.
++
+image::gateway-quickstart-add-gateway.png["Screenshot of 'Add a Flex Gateway'",width=90%]
+
+. Run the copied code block in your terminal window, making sure to replace `<gateway-name>` with a name, such as `my-gw`.
++
+[NOTE]
+Use `sudo` if you encounter file permission issues when running this command.
+
+. Select *Flex Gateways* in the side navigation panel.
++
+Your registered Flex Gateway now appears in the list of gateways.
+
+== Run Flex Gateway
+
+To run Flex Gateway, run the following command:
+
+[source,ssh]
+----
+docker run --rm \
+  -v "$(pwd)":/usr/local/share/mulesoft/flex-gateway/conf.d \
+  -p 8081:8081 \
+  mulesoft/flex-gateway
+----
+
+[NOTE]
+Use `sudo` if you encounter file permission issues when running this command.
+
+A new Flex Gateway instance is now running in your terminal window.
+
+== Publish a Simple API
+
+. Select *API Manager* from the Anypoint Platform menu.
+. Click *Add API*, followed by *Add new API*.
++
+image::gateway-quickstart-add-new-api.png["Screenshot of 'Add new API'",width=35%]
+
+. In the *Add API* page, select *Flex Gateway* as your runtime.
+. Select your registered gateway in the *Select a gateway* section.
+. Click *Next*.
++
+The following screenshot of the *Add API* page highlights the specified options.
++
+image::gateway-quickstart-select-gateway-for-api.png["Screenshot of 'Select Gateway for API'",width=100%]
+
+. Click *Create New API*.
+. Specify an API name and select *HTTP API* as the asset type.
+. Click *Next*.
+. In the *Endpoint* page, enter the following for *Implementation URI*:
```

Comment:
is this a field? Maybe enter xx in the xx field?

Reply:
Fixed